.changelog/322.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+`cloudavenue_vm` - Add DataSource.
+```
```

Comment:
Hashicorp recommand this syntax 
```suggestion
```release-note:new-datasource
`datasource/cloudavenue_vm` - Short description.
```
```